### ISSUE TITLE
Status of DistillKit updates?

### ISSUE THREAD

**Issue by misdelivery** [arcee-ai/DistillKit#27]
Hi,
The README for the [Llama-405B-Logits](https://huggingface.co/datasets/arcee-ai/LLama-405B-Logits) dataset mentions that the DistillKit repository would "soon be updated with proper training scripts and additional resources." However, it's been over four months, and there don't appear to be any updates yet.

Could you share any updates or timeline on when these resources might be available?

Thanks!
